### ISSUE TITLE
Respect Fitts's Law when in full screen

### ISSUE THREAD

**Issue by nick-s-b** [mpv-player/mpv#9791]
### Expected behavior of the wanted feature

Hi all,

I find it that `mpv` in full screen mode does not take advantage of "magic corners" and Fitts's Law. In a nutshell, when you quickly move your mouse to one of the corners (mainly the top left or right (to close the window) or to bottom left (to access play/pause buttons), the UI overlay does get activated **but you cannot press the buttons or perform actions that are in those corners.**

Wikipedia description of Fitts's Law describes it as: https://en.wikipedia.org/wiki/Fitts's_law

>Fitts law (often cited as Fitts' law) is a predictive model of human movement primarily used in human–computer interaction and ergonomics. This scientific law predicts that the time required to rapidly move to a target area is a function of the ratio between the distance to the target and the width of the target.

And "magic corners are"
>This effect can be exaggerated at the four corners of a screen. At these points two edges collide and form a theoretically infinitely big button. Microsoft Windows (prior to Windows 11) places its "Start" button in the lower left corner and Microsoft Office 2007 uses the upper left corner for its "Office" menu. These four spots get called the “magic corners”. [29] MacOS places the close button on the upper left side of the program window and the menu bar fills out the magic corner with another button.

For example, if I move my mouse to top left corner of the screen when in full screen mode, I see this:

![2022-01-28-211856_3840x2160_scrot](https://user-images.githubusercontent.com/7957590/151643557-9f267930-75f3-4653-8424-930f04b93f94.png)

I should be able to click and "close window" should be activated. But it is not. I have to move the pointer few pixels to the right and down to actually click the close button. That requires fine motor movement and takes time. Same when I move my mouse pointer to bottom left:
![2022-01-28-211924_3840x2160_scrot](https://user-images.githubusercontent.com/7957590/151643600-47614df6-498d-47e3-8ce4-6e02a198c3ef.png)
Again, I cannot activate the play/pause button because the UI does not recognize a left bottom corner as clickable.

Making buttons in corners clickable would make the `mpv` a lot easier and faster to use and would minimize the need for micro-movements and adjustments. That would also benefit disabled people.

**Comment by garoto** [comment]
I thought teaching kids enrolled in CS that maximizing pixel-hunting when designing was the standard procedure. Because oh boy, this shenanigan has being going for a fricking long time.

Glad that with mpv I can use a keyboard for mostly anything.

**Comment by lextra2** [comment]
I agree with OP. But I also want to say, if you want to maximize speed, get used to keyboard shortcuts.
